src/SignalR/clients/ts/FunctionalTests/webpack.config.js

```diff
@@ -23,24 +24,35 @@ module.exports = {
             }
         ]
     },
-    resolveLoader: {
-        // Special resolution rules for loaders (which are in the 'common' directory)
-        modules: [ path.resolve(__dirname, "..", "common", "node_modules") ],
-    },
     resolve: {
         extensions: [".ts", ".js"]
     },
+    externals: {
+        "@microsoft/signalr": "signalR",
+        "@microsoft/signalr-protocol-msgpack": "signalR.protocols.msgpack",
+        "signalR": "signalR",
```

Comment:
What's this?

Reply:
All the symbols that are defined externally (signalR is defined on the window) so this tells webpack to avoid resolving it.